src/main/java/de/tum/in/www1/artemis/repository/ExerciseGroupRepository.java

```diff
@@ -15,6 +16,9 @@
 @Repository
 public interface ExerciseGroupRepository extends JpaRepository<ExerciseGroup, Long> {
 
+    @Query("SELECT e FROM ExerciseGroup e WHERE e.exam.id = :#{#examId}")
```

Comment:
We don't need the @query here, as the method name should suffice.
See https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods